src/transform.rs

```diff
@@ -69,21 +67,47 @@ pub struct Decomposed<V: VectorSpace, R> {
     pub disp: V,
 }
 
-impl<P: EuclideanSpace, R: Rotation<P>> Transform<P> for Decomposed<P::Diff, R>
+impl<P: EuclideanSpace, R: Rotation<Space = P>> One for Decomposed<P::Diff, R>
 where
     P::Scalar: BaseFloat,
-    // FIXME: Investigate why this is needed!
     P::Diff: VectorSpace,
```

Comment:
Can we remove this bound here?

Reply:
It's needed because there's a lot of vector-scalar multiplication in the implementation.

Reply:
This is just a bound on `One` not on `Mul` right? It seems there are only vector-scalar multiplications in the `Mul` implementation.

Reply:
Oops, I was looking at the wrong line.